OpenTESArena/src/Entities/Camera3D.cpp

```diff
@@ -6,8 +6,8 @@
 #include "../Math/Constants.h"
 #include "../Math/Quaternion.h"
 
-Camera3D::Camera3D(const Double3 &position, const Double3 &direction)
-	: position(position), forward(direction)
+Camera3D::Camera3D(const Double3 &direction, const Double3 &position)
```

Comment:
`position` is first again in the parameters, but I initialize `forward,` `right` and `up` first to match the order they are declared to satisfy the warning. Is this all right?